### DISCUSSION TITLE
When using translateY, elements on Android become invisible after the 4th item in the list.

### DISCUSSION THREAD

**Discussion by EvgenMart** [software-mansion/react-native-reanimated#6862] [Q&A] [answered]
`When` using translateY, elements on Android become invisible after the 4th item in the list.

For iOS, everything works as expected.

android 

https://github.com/user-attachments/assets/5a148ef0-3818-45e1-8945-69653f0cfca0

iOS

https://github.com/user-attachments/assets/0ea5f6f7-ec87-43d1-b918-2108e14bf9e6

styles

const ITEM_HEIGHT = 150;

  const animatedStyle = useAnimatedStyle(
() => {
    const translateY = withTiming(
      interpolate(
        pullDownPosition.value,
        [0, ITEM_HEIGHT * 2],
        [0, -index * ITEM_HEIGHT],
        Extrapolation.CLAMP
      ),
      {
        duration: 150,
        easing: Easing.out(Easing.ease),
      }
    );
    return {
      transform: [{ translateY }],
      zIndex: 1000 - index,
      elevation: 1000 - index,
    };
  }, [pullDownPosition]);

I would greatly appreciate your help.

**Comment by MatiPl01** [comment, marked as answer]
Hey @EvgenMart!

Have you tried adding this prop `removeClippedSubviews={false}` to the `FlatList` component? If it doesn't help, can you possibly share the entire implementation to test it out?

**Reply by EvgenMart** [reply]
@MatiPl01, Thank you very much. That helped.
